ground/src/main/proto/com/google/ground/geometry.proto

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+option java_package = "com.google.android.ground.proto";
+option java_multiple_files = true;
+
+// An arbitrary geometry, represented using WGS 84 Web Mercator coordinate
+// system. Designed for compatibility with Google Maps Platform and GeoJSON
+// format.
+message Geometry {
+    // A geometry may be one of the following types:
+    // Required.
+    oneof geometry_type {
+        // A point geometry.
+        Point point = 1;
+        // A polygon geometry.
+        Polygon polygon = 2;
+        // A multi-polygon geometry.
+        MultiPolygon multi_polygon = 3;
+    }
+}
+
+// A pair of coordinates in the WGS 84 Web Mercator system.
+message Coordinates {
+    // Required. WGS 84 latitude.
+    double latitude = 1;
+    // Required. WGS 84 longitude.
+    double longitude = 2;
+}
+
+// A single point on the map.
+message Point {
+    // Required. The point's coordinates..
+    Coordinates coordinates = 1;
+}
+
+// A linear ring geometry.
+message LinearRing {
+    // Required. The coordinates of the ring's vertices.
```

Comment:
Maybe worth mentioning that the first and last coordinate should be identical. Also below.